packages/design-system/src/components/Banner/Banner.js

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+
+import styles from './style.scss';
+import { BannerIcon } from './BannerIcon';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'banner';
+
+export const Banner = props => {
+  const {
+    className,
+    onClose,
+    size,
+    text,
+    type,
+    ...restProps
+  } = props;
+
+  const mergedWrapperClassNames = getMergedClassNames(
+    cx({
+      [`${baseClass}__close`]: true,
+      [`${baseClass}__content--${size}`]: size,
```

Comment:
why it's used here and in the line 40 (in the child element)? something is still wrong here